pallets/msa/src/tests.rs

```diff
@@ -2021,6 +2018,31 @@ fn register_signature_and_validate(
 	assert_eq!(Some(mortality_block as u64), actual);
 }
 
+#[test]
+// for illustration purposes
+pub fn bucket_for() {
```

Comment:
I don't think this should have moved. Rebase issue?

Reply:
Yes plenty rebase, let me move them around